translator/internaldata/resource_to_oc.go

```diff
@@ -23,11 +23,45 @@ import (
 	occommon "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
 	ocresource "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
 	"github.com/golang/protobuf/ptypes"
+	"go.opencensus.io/resource/resourcekeys"
 
 	"go.opentelemetry.io/collector/consumer/pdata"
 	"go.opentelemetry.io/collector/translator/conventions"
 )
 
+type ocResourceTypeDetector struct {
```

Comment:
The word `Detector` is overused in the context of resources. How about `ocResourceTypeMapper`?

Reply:
Renamed to `ocInferredResourceType` to make its purpose more explicit, feel free to suggest a better name :)